commands.py

```diff
@@ -134,7 +134,7 @@ def run(self, edit: sublime.Edit, completion_index: int) -> None:
         # We don't have to care whether it's an inline completion or not.
         source_line_region = self.view.line(sublime.Region(*completion["region"]))
         self.view.erase(edit, source_line_region)
-        self.view.insert(edit, source_line_region.begin(), completion["completionText"])
+        self.view.insert(edit, source_line_region.begin(), completion["displayText"])
```

Comment:
The tricky part is that I think `displayText` shows how it should looks like for the "whole" block. Not only those lines after the current line. While `completionText` is those lines after the current line, which is for insertion imo.

Reply:
So you have an example screenshot of this ? I've never experienced it

Reply:
Hmm... not sure what is supposed to do in this case honestly.
![image](https://user-images.githubusercontent.com/6594915/180605391-1c6c208a-1a39-4a96-bb6f-25a601717047.png)

Reply:
That is a weird case, we should prevent panelcompletions if the region / line where the cursor is empty as a work around until I see how VSCode handles these